www_src/lib/keyboard.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  simulateTabPress: function () {
+    var current = document.activeElement;
+    var elements = document.getElementsByTagName('input');
```

Comment:
technically, not enough. `<select>` and `<textarea>` and friends also allow tabindex for cycling through. `document.querySelectorAll('input, select, textarea, button, a, iframe');` if we want to be *really* complete, but leaving off "a" and "iframe" probably makes sense.